moveos/moveos-stdlib/rooch-framework/sources/account.move

```diff
@@ -1,4 +1,4 @@
-module mos_framework::account{
+module rooch_framework::account{
```

Comment:
after rename the new directory structure is:

-moveos-stdlib
| --moveos-stdlib
| --rooch-stdlib

The positioning of moveos is an independent library. Maybe it should be that rooch-framework depends on moveos-stdlib. Now moveos-stdlib directly includes rooch-framework, and the dependency relationship is a bit strange.

Think of two optimization methods, according to the existing directory structure, rename to
-framework
|--moveos-stdlib
|--rooch-framework

Or the rooch-framework is raised one level up, and it is at the same level as moveos.

Reply:
I will move out the `rooch-framework` from `moveos`, but need to refactor the framework build.